pom.xml

```diff
@@ -63,6 +63,27 @@
           <version>1.3</version>
           <scope>test</scope>
         </dependency>
+
+        <!-- Workflow plugin imports are used to test compatibility -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>1.8</version>
```

Comment:
Makes sense as the version is repeated 3 times, as long as you are releasing all three of those plugins at the same time. If/when you refactor them into separate git repositories with independent release lifecycles then it would make sense to list explicit versions... but for now a property will ensure consistent versioning